examples/pytorch/resnet.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: jobset.x-k8s.io/v1alpha1
```

Comment:
revert all files but this one since they are merged in the other pr

Reply:
done